components/apimgt/org.wso2.carbon.apimgt.hostobjects/src/main/java/org/wso2/carbon/apimgt/hostobjects/APIStoreHostObject.java

```diff
@@ -5156,12 +5157,16 @@ public static boolean jsFunction_hasUserPermissions(Context cx, Scriptable thisO
                     if ("DisplayName".equalsIgnoreCase(dto2.getPropertyName())) {
                         displayName = dto2.getPropertyValue();
                     }
+                    if ("DisplayOrder".equalsIgnoreCase(dto2.getPropertyName())) {
+                        displayOrder = Integer.parseInt(dto2.getPropertyValue());
```

Comment:
can dto2.getPropertyValue() be null or empty string?

Reply:
fixed